checks/E215A6.yaml

```diff
@@ -0,0 +1,62 @@
+id: "E215A6"
+name: SAPHanaFilesystem resource configured
+group: Pacemaker
+description: |
+  SAPHanaFilesystem resource is configured
+remediation: |
+  ## Abstract
+  SAPHanaFilesystem resource should be configured especially for hana_scale_out clusters. So, that problems with HANA filesystem can be monitored and remediative steps can be taken by cluster if the resource is hung due to the fileystem unavailability.
+  ## Remediation
+
+  Configure SAPHanaFilesystem resource.
+
+  ## References
+  AZURE:
+
+    - https://learn.microsoft.com/en-us/azure/sap/workloads/sap-hana-high-availability?tabs=lb-portal%2Csaphanasr-angi
+
+  AWS:
+
+    - https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-angi-scaleout-perfopt-15/index.html#id-saphanafilesystem
+
+  GCP:
+
+    - https://cloud.google.com/solutions/sap/docs/sap-hana-ha-scaleout-config-sles#create_the_saphanafilesystem_primitive_resource
+
+  SUSE / KVM:
+
+    - https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-angi-scaleout-perfopt-15/index.html#id-saphanafilesystem
+
+  Nutanix:
+
+    - https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-angi-scaleout-perfopt-15/index.html#id-saphanafilesystem
+
+severity: warning
+metadata:
+  target_type: cluster
+  cluster_type: 
+    - hana_scale_up
+    - hana_scale_out
+  architecture_type: angi
+
+facts:
+  - name: resource_filesystem
+    gatherer: cibadmin@v1
+    argument: cib.configuration.resources
+
+values:
+  - name: resource_type
+    default: "SAPHanaFilesystem"      
```

Comment:
I think this value should include `customization_disabled: true  `.
It is a hardcoded value